hack/patch_node.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+PROXY_PORT=9999
+
+function finish {
+  rc=$?
+  if jobs 1 | grep -q Running; then
+    echo
+    echo "Killing proxy (pid=$proxy_pid)..."
+    kill "$proxy_pid"
+  fi
+  exit "$rc"
+}
+
+# creates/removes prow/sriov resources for the given $node
+function patch_node {
+  local node=$1
+  local capacity=$2
+  if [ "$capacity" -ne 0 ]; then
```

Comment:
Can someone pass  negative value to `capacity` ? 
Maybe change to `if [ "$capacity" -ge 0 ]; then`  ?

Reply:
negative values should not be used.
you meant `-gt` i guess, we can bu tthen we would need to protect against non integers, which -ne do protect them
and i wanted to keep it simple as was suggested on other PRs.

i can add something like in the main, wdyt?
```
  if [ -z "${capacity##*[!0-9]*}" ]; then
    echo "error: capacity $capacity must be greater or equal 0"
    exit 1
  fi
```

Reply:
I think it is better to play it safe.